package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "back-end-project-week",
+  "version": "1.13.0",
+  "main": "index.js",
+  "repository": "https://github.com/jalvarez2020/back-end-project-week.git",
+  "author": "Javier Alavrez <javieralvarez82@live.com>",
+  "license": "MIT",
+  "dependencies": {
+    "axios": "^0.18.0",
+    "bcryptjs": "^2.4.3",
+    "cors": "^2.8.5",
+    "express": "^4.16.4",
+    "helmet": "^3.15.1",
+    "jsonwebtoken": "^8.4.0",
+    "knex": "^0.16.3",
+    "sqlite3": "^4.0.6"
+  },
+  "scripts": {
+    "server": "nodemon start"
```

Comment:
I would add in a start script for production/deployment it should say something like this:
`node index.js`
since nodemon is only for development.

Reply:
Added start script as advised. Thank you